packages/light-apps/package.json

```diff
@@ -26,6 +26,7 @@
     "@substrate/ui-common": "^0.2.0",
     "@substrate/ui-components": "^0.2.0",
     "@types/file-saver": "^2.0.0",
+    "@types/filesystem": "^0.0.29",
```

Comment:
n.b. this was added to quiet tsc complaining about a nonexistent triple slash directive.